src/index.ts

```diff
@@ -29,10 +29,14 @@ export class Uwuifier {
   @InitModifierParam()
   private _exclimationsModifier: number = 1;
 
-  constructor(spacesModifierPara?: spacesModifier, wordsModifierPara?: number, exclimationsModifierPara?: number) {
-    if (typeof spacesModifierPara !== 'undefined') this._spacesModifier = spacesModifierPara;
-    if (typeof wordsModifierPara !== 'undefined') this._wordsModifier = wordsModifierPara;
-    if (typeof exclimationsModifierPara !== 'undefined') this._exclimationsModifier = exclimationsModifierPara;
+  constructor({
+    spacesModifierParam = { facePercentage: 0.05, actionPercentage: 0.05, stutterPercentage: 0.1 },
+    wordsModifierParam = 1,
+    exclimationsModifierParam = 1
+  }) {
```

Comment:
Are these still optional though? And what happens if I for example put in 0 or null for action percentage. Maybe we should write a test for these different cases aswell.